### PR TITLE
Add basic info 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- There is no connection to the backend. #1 
- There is no text helper to replace variables with user data. #2
- There is no custom client personalization. #3

**What is the new behavior (if this is a feature change)?**
- Connection to the backend has been established. #1 
- A text helper component has been implemented. #2
- The client's logo, icon and title has been added. #3

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks good to me.

File: src/components/helpers/Text/index.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    getPureText = (text) => {
+        let rx = /\{(.*?)\}/;
+        let res = text.replace(rx, "");
+        return res;
+    }
+
+    printText = () => {
+        let text = this.props.value;
+        if(text !== null && text !== undefined){
+            if(localStorage.getItem('f_n') && localStorage.getItem('f_l') && localStorage.getItem('f_e') && localStorage.getItem('f_p')){
```
Comment (simonprast) [inline src/components/helpers/Text/index.jsx]: Line length :)

**Review by simonprast — CHANGES_REQUESTED** [review]
Code quality looks good overall, one line was too long though.

File: src/components/helpers/Text/index.jsx (hunk 1 above)
Comment (Aichnerc) [reply src/components/helpers/Text/index.jsx]: Fixed that 🦋

**Review by simonprast — APPROVED** [review]
Nice

**Pull request merged**